tests/tests/basic_scenarios.robot

```diff
@@ -94,7 +85,8 @@ Add multiple files and remove one
     Should Be Equal As Numbers    ${count_after_removal}    ${count_added-1}
 
 Load and reload file
-    FOR    ${test_file}    IN    @{TestFiles}
+    FOR    ${file}    IN    @{TestFiles}
+        ${test_file}    Set Variable    ${TestFiles}[${file}]
```

Comment:
It might be useful to use "Set Test Variable" instead of normal "Set Variable" so other tests wont be able to use/overwrite it.

Reply:
Agreed with Krzysztof that 'Set variable' topic will be handle by separate PR.